packs/data/spell-effects.db/spell-effect-mantle-of-the-frozen-heart-icy-claws.json

```
@@ -27,7 +27,7 @@
                 },
                 "group": "brawling",
                 "key": "Strike",
-                "label": "PF2E.Weapon.Base.claw",
+                "label": "PF2E.BattleForm.Attack.IcyClaw",
```

```suggestion
                "label": "PF2E.Weapon.Base.claw",
```
`You gain a claw unarmed attack`